extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/runtime/QuartzScheduler.java

```diff
@@ -110,6 +110,10 @@ public void startTimer(long delay, Runnable action) {
     }
 
     void start(@Observes StartupEvent startupEvent) {
+        if (schedulerConfig.getSchedules().isEmpty()) {
+            LOGGER.warn("No Schedules found, Scheduler will not be started.");
```

Comment:
```suggestion
            LOGGER.warn("No @Scheduled annotations found, Scheduler will not be started.");
```

@mkouba that being said, I wonder if it should really be a warn?

Reply:
I believe so. The extension should be removed if there is no scheduled method...

Reply:
Unless we provide a way to schedule a job programatically...

Reply:
@SlyngDK could you apply my wording suggestion and squash everything into a single commit? Thanks!

Reply:
Actually, it should probably read `No @Scheduled methods found`...

Reply:
Whatever makes you happy :)